designing-event-driven-processes/README.md

```diff
@@ -0,0 +1,18 @@
+# Beyond basic messaging: Designing long-running processes
+
+## Abstract
+
+Messaging in distributed systems offers a range of valuable benefits: it enables the design of decoupled components, where each component can evolve and scale individually. In addition, messaging improves the system’s fault tolerance but goes even further by opening the door to building self-healing systems. One benefit that doesn’t usually appear in the Googleable list of benefits is one of the most powerful: supporting real-life, long-running business processes and making the system mirror real-world business use cases. And that’s exactly the focus of this workshop.
```

Comment:
The way it is stated now reads as if messaging is the only way, and it'll do automagically, design decoupled and scalable components. What about the following:

I also tried to remove the "but."

``` suggestion
Messaging in distributed systems offers a range of valuable benefits, including the design of decoupled components, where each component can evolve and scale independently. In addition to improving the system’s fault tolerance, it opens the door to building self-healing systems. One benefit that doesn’t usually appear in the Googleable list of benefits is one of the most powerful: supporting real-life, long-running business processes and making the system mirror real-world business use cases. And that’s precisely the focus of this workshop.
```

Reply:
Mm, that's why I added "it enables", to counter the thought that it automagically happens. Funny thing though, I have exactly this point in one of my sessions 🤣 I really like all the other edits, but don't you think keeping the "it enables" is better though?